frontend/packages/client/src/components/Header.js

```diff
@@ -18,7 +19,8 @@ function Header(props) {
   return (
     <>
       <header
-        className={`header ${props.location.pathname} has-background-white is-block navbar is-fixed-top`}
+        id="navbar"
```

Comment:
added this id as the `.header` class felt too generic for the specific style rule of the 4px yellow border